MuneysCoin/Dashboard
# Improvement/swapping enhancements (#9)

* initial dashboard layout (#1)

* initial dashboard layout

* setup stuff for web3

* responsive menu

* metamask connect

* publish to cloudflare pages

* setup deployments

* deployment stuff

* remove workflow

* trigger dep

* remove space

* Feature/web3 (#3)

* wallet connect instead of web3

* test

* some initial setup stuff

* add check for client

* small tweak

* only allow bsc

* change modal title

* fixes

* no useweb3modal

* fix lifecycle

* another lifecycle fix

* finished implementing wallet connection

---------



* basic layout swapping card (#4)

* basic layout swapping card

* add testnet support

* get tokens

* setup stuff for toke select

* improved selection mechanism

* fix width swapping card

* fix default selection

* swap the swappers

* fix mobile stuff

* put back default tokens

* amount added

* swapping implemented

* get prices in usd

* swapping complete

---------



* skip validation (#5)

* fix early modal (#6)

* add validation back (#7)

* wallet check

---------
